pkg/postage/stamp.go

```diff
@@ -118,6 +119,35 @@ func (s *Stamp) UnmarshalBinary(buf []byte) error {
 	return nil
 }
 
+type stampJson struct {
```

Comment:
I'd also add JSON tags on the fields, in case someone will rename the fields in the future.